sumpy/tools.py

```diff
@@ -862,4 +911,18 @@ def matvec_toeplitz_upper_triangular(first_row, vector):
         output[row] = sym.Add(*terms)
     return output
 
+
+def to_complex_dtype(dtype):
+    if dtype == np.complex64:
+        return np.complex64
+    if dtype == np.complex128:
+        return np.complex128
+    if dtype == np.float32:
+        return np.complex64
+    if dtype == np.float64:
+        return np.complex128
+    raise RuntimeError(f"Unknown dtype: {dtype}")
```

Comment:
`dict`?

Reply:
Fixed in 2d06371313e990320852024477a70e9c535c6142